<?xml version="1.0"?>

<launch>
	<node name="KeyboardManager" pkg="human_aware_navigation" type="KeyboardManager"/>
</launch>
